packages/app/src/components/Blank.cy.tsx

```diff
@@ -1,55 +1,42 @@
-import { initial, initialCT, session, sessionLifecycle, visitFailure } from './Blank'
+import { initial, session, sessionLifecycle, visitFailure } from './Blank'
 import { getContainerEl } from '@cypress/mount-utils'
 
-describe('initial - e2e', () => {
+describe('initial', () => {
   beforeEach(() => {
     getContainerEl()!.innerHTML = initial()
   })
 
   it('works', () => {
     cy.percySnapshot()
   })
-
-  it('links to docs', () => {
-    cy.contains('cy.visit').should('have.attr', 'href', 'https://on.cypress.io/visit')
-  })
 })
 
-describe('initial - ct', () => {
+describe('session', () => {
   it('works', () => {
-    getContainerEl()!.innerHTML = initialCT()
+    getContainerEl()!.innerHTML = session()
```

Comment:
We've talked about turning percy off till we can get it stable. Is there any way we can bulk up these tests?

Reply:
I think the percy tests makes the most sense. AFAIK, we are just commenting out the percy snapshots that are failing, not turning off percy completely.

Reply:
I was under the impressing we were turning them off. 🤷🏻‍♀️ we dont' have a solid way to verify this behavior locally though since percy only runs on Ci

Reply:
I'll update the tests to verify the text.

Reply:
In the doc that was merged it settled on just skipping specific Percy tests that are problems, not turning it all off. 

> we dont' have a solid way to verify this behavior locally though since percy only runs on Ci

There's something that helps here, and seems to be broken for component testing, and we should look into it - the custom `cy.percySnapshot()` command is supposed to log a pinnable Cypress snapshot of the exact DOM state that will be stringified and sent to Percy. So we can see locally what Percy will receive and shouldn't technically need to wait on CI except to verify.

Reply:
(I'm not actually reviewing this PR but fwiw these pages look great)

Reply:
verified the text: [9bb31cf](https://github.com/cypress-io/cypress/pull/23095/commits/9bb31cf7ba7472b0b2371d1ea90ebbe8fbe93002)